Applications/AlgorithmPlaza.qmd

```diff
@@ -6,37 +6,41 @@ aliases:
+
+Within the platform, the hosted algorithms can be used through user interfaces or APIs. Users can seamlessly onboard their algorithms on the openEO Algorithm Plaza for further exposure to a large audience.
 
 ## Overview 
 
-A captivating feature of the marketplace is the growing and diverse catalogue of EO services from different providers. To enhance user experience when searching a service, a text filter bar is available at the top of the page in addition to attribute filtering. 
+The marketplace showcase a diverse catalogue of EO services from various providers. These services are classified based on their maturity levels, which indicate the quality and reliability of the service.
 
 ![](_images/PlazaOverview.png)
 
-Each service, represented as a placard in the above image, has its dedicated page providing detailed information. This page briefly describes the methodology, expected results, and instructions for executing the service.
+Each service has its dedicated page providing detailed information. The information includes the methodology, expected results, and service execution instructions.
 
 
 ## What is a service?
 
-openEO Algorithm Plaza offers a wide range of services in Earth Observation. These services support algorithms ranging from simple computations like the Normalized Difference Vegetation Index (NDVI) to more complex algorithms that utilize machine learning and multiple parameters. 
+openEO Algorithm Plaza offers a wide range of workflows in Earth Observation termed as services. These services can ranging from simple computations like the Normalized Difference Vegetation Index (NDVI) to more complex algorithms that utilize machine learning and multiple parameters. 
 
-In addition to providing existing services, the marketplace also supports users in showcasing their algorithms as services in its catalogue. To advertise your service on the marketplace, the algorithm must be built using openEO. It's important to consider your target audience, especially if reaching a non-scientific audience; you may want to hold back on hard-to-interpret options.
+In addition to providing access to available services, the marketplace also supports users in showcasing their algorithms. To advertise an algorithms on the marketplace, they must be built using openEO. Visit the openEO [UDP](https://open-eo.github.io/openeo-python-client/udp.html){target="_blank"} for more information on openEO user-defined-processes.
 
-Once your algorithm is exposed as a service, users can 'invoke'  it with a given set of parameters. 
+Once an algorithm is exposed as a service, other users readily access and use it for their own workflows. To enhance reusability, it's essential to consider the audience when preparing materials. Especially for non-scientific users, it's advisable to avoid complex options that may be difficult to understand.
```

Comment:
I would leave: "Especially for non-scientific users, it's advisable to avoid complex options that may be difficult to understand."" out